src/options.js

```diff
@@ -0,0 +1,41 @@
+const fs = require("fs");
+
+export const prepareOptions = (optionsFromCLI) =>
+  Object.keys(optionsFromCLI).reduce((acc, optionKey) => {
+    if (optionKey == "version" || typeof optionKey == "object") {
+      return acc;
+    }
+
+    const value = optionsFromCLI[optionKey];
+
+    if (optionKey.startsWith("margin")) {
+      const key = optionKey.replace("margin", "");
+      return {
+        ...acc,
+        margin: {
+          ...acc.margin,
+          [key]: value,
+        }
+      }
+    }
+
+    if (
+        ["headerTemplate", "footerTemplate"].includes(optionKey)
+        && value.startsWith("file://")
+    ){
+      const loadedFile = fs.readFileSync(
+        value.replace("file://", ""),
+        "utf-8"
+      );
+      return {
+        ...acc,
+        [optionKey]: loadedFile,
+      }
+    }
+
+    return {
+      ...acc,
+      [optionKey]: value,
+    };
```

Comment:
Why do we need the redundant code?

Reply:
I decided to keep it for now so I will know if the only update of version will break something or maybe my refactoring. I will clean this up in the step of replacing QT webkit 👍